src/graphql/root/subscription/sat-price.ts

```diff
@@ -0,0 +1,124 @@
+import crypto from "crypto"
+
+import { SAT_PRICE_PRECISION_OFFSET } from "@config"
+
+import { Prices } from "@app"
+
+import { checkedToDisplayCurrency, DisplayCurrency } from "@domain/fiat"
+import { customPubSubTrigger, PubSubDefaultTriggers } from "@domain/pubsub"
+
+import { GT } from "@graphql/index"
+import { mapError } from "@graphql/error-map"
+import { UnknownClientError } from "@graphql/error"
+import PricePayload from "@graphql/types/payload/price"
+
+import { PubSubService } from "@services/pubsub"
+import { baseLogger } from "@services/logger"
+
+const pubsub = PubSubService()
+
+const SatPriceInput = GT.Input({
+  name: "SatPriceInput",
+  fields: () => ({
+    priceCurrencyUnit: { type: GT.NonNull(GT.String) },
```

Comment:
`btcPrice` is deprecated... we cant use the same types because currencies are dynamic (not easy way to build a dynamic graphql schema without have price server query as a dependency)

Reply:
because this is currency unit and in btcPrice is only currency...  do you think is better to change it to currency?

Reply:
Fixed according to our offline chat  https://github.com/GaloyMoney/galoy/commit/2974fb4e6e9b68f42d7d6432788b9c3bfde72c85